### PR TITLE
go script and routing

### PR DESCRIPTION
**Pull request by joshmccall221**

### CODE REVIEW

File: src/views/App/App.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ class App extends Component {
           <Route path={HOME_URL} component={MainScreen} />
           <Route path={CONTENT_URL} component={ContentScreen} />
           <Route path={ABOUT_URL} component={AboutScreen} />
+          <Route component={MainScreen} />
```
Comment (joshmccall221) [inline src/views/App/App.js]: Probably a better way but the cordova has trouble with routing.  This hack is basically serving the mainScreen as the 404 page and from there the routing works as expected. ie. no white screen. 👍

File: go.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#npm install
+#rm -rf corber
+#sed -i '/corber/d' ./package.json
+#corber init
+#corber platform remove android
+#corber platform add android
+#corber proxy requirements
+#corber serve --platform=android
+corber build --platform=android
+sed -i 's%href="/%href="%g' ./corber/cordova/www/index.html
```
Comment (joshmccall221) [inline go.sh:50]: again... routing ...

File: src/views/App/App.js (hunk 1 above)
Comment (drewdiddy611) [reply src/views/App/App.js]: Should make this render the splash screen as that has the redirect if they've already been through them.

Comment (aiguy611) [reply src/views/App/App.js]: Should make this render the splash screen by default as it has the redirect if they've already viewed them.

[Refer to this](https://github.com/drewdiddy611/yffr-app/blob/546ba27f53003037321472ebfffb195eff0086bb/src/views/pages/SplashScreen/SplashScreen.js#L16)

**Review by aiguy611 — CHANGES_REQUESTED** [review]

**Review by aiguy611 — APPROVED** [review]

**Pull request merged**